### PR TITLE
v1.25.7 – Added new courier links for NZ

### PR DESCRIPTION
**Pull request by markedup**
Additional 2 links for the NZ tenant data. No changes to existing layout.

### CODE REVIEW

File: src/templates/resources/footer.json [hunk 1]
```diff
@@ -1231,6 +1229,14 @@
                 "url": "https://connect.menulog.co.nz/login?client_id=RappsOrigPartnerCentre&response_type=code&redirect_uri=https:%2F%2Fpartner.menulog.co.nz%2Fsignin-jeconnect&state=QaF4ae5nfj7FUpuoevTVCHvmMBxIA1hO_4mDtePNEJOasCly6wdTLaHjTuvS7_If4SwPYZelgwT0ZFYb7CCZvfIIr9X0OVLKiRpdCLYU9zJPuorMgnFjjRlyQNdU8L_5ZvkfGGBtXJx-jNOlh5lU_aqiQ-KgrKthGa83ZbYH8GjV81bwGb7iJ8PRFJPYbwTyys-zclUwNFxXhfdsOpqIWsIlfpQ",
                 "text": "Partner Centre"
             },
+            {
+                "url": "https://couriers.menulog.co.nz/application",
```
Comment (kevinrodrigues) [inline src/templates/resources/footer.json]: Should these urls work?

Comment (markedup) [reply src/templates/resources/footer.json]: - `/login` was working for me yesterday.
- `/application` seemed to be a redirect back to our `just-eat.com` domain.

**Comment by kevinrodrigues** [comment]
@markedup PR branches should be named with the version your releasing. e.g `v.1.2.3-nz-courier-links` commits would include that version name then a description.

**Comment by markedup** [comment]
> @markedup PR branches should be named with the version your releasing. e.g `v.1.2.3-nz-courier-links` commits would include that version name then a description.
Apologies, will look to rectify.

File: src/templates/resources/footer.json (hunk 1 above)
Comment (kevinrodrigues) [reply src/templates/resources/footer.json]: Weird, doesn't work for me at all.

**Comment by DamianMullins** [comment]
@markedup is this PR still required?

**Pull request closed without merging**